content/en/docs/concepts/storage/volumes.md

```diff
@@ -1302,16 +1302,15 @@ persistent volume:
 
 #### CSI raw block volume support
 
-{{< feature-state for_k8s_version="v1.14" state="beta" >}}
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
 Starting with version 1.11, CSI introduced support for raw block volumes, which
 relies on the raw block volume feature that was introduced in a previous version of
 Kubernetes.  This feature will make it possible for vendors with external CSI drivers to
 implement raw block volumes support in Kubernetes workloads.
 
-CSI block volume support is feature-gated, but enabled by default. The two
-feature gates which must be enabled for this feature are `BlockVolume` and
-`CSIBlockVolume`.
+This feature requires `CSIBlockVolume` feature gate to be enabled. It
+is enabled by default starting with Kubernetes 1.14.
```

Comment:
Usually, stable features _cannot_ be disabled. Is that the case here?

Reply:
Yes, it's the case. Removed whole paragraph.